jobs/commons/src/services/interop-token.service.ts

```diff
@@ -0,0 +1,62 @@
+import { KMSClient, SignCommand, SignCommandInput } from '@aws-sdk/client-kms'
+import { TokenGenerationConfig } from '../config/interop-token.config.js'
+import { InteropJwtHeader, InteropJwtPayload, InteropToken, JWT_INTERNAL_ROLE, b64ByteUrlEncode, b64UrlEncode } from '../index.js'
+import { v4 as uuidv4 } from 'uuid'
```

Comment:
Just for information: Node has an internal module named `crypto` that can be used to generate UUIDs without relying in external packages.

```typescript
import crypto from "crypto"
const uuid = crypto.randomUUID()
```

Reply:
nice, I'll use that instead, thanks!